app/src/main/java/com/android/streetworkapp/model/moderation/PerspectiveAPIModels.kt

```diff
@@ -0,0 +1,107 @@
+package com.android.streetworkapp.model.moderation
+
+import kotlinx.serialization.Serializable
+
+// All the definitions below are used to interact with Perspective API
+
+/** Default values we can use as thresholds for our TextModerationTags */
+object PerspectiveAPIThresholds {
+  val DEFAULT_THRESHOLD_VALUES = TextModerationTags.entries.associateWith { 0.2 }
+}
+
+/** All the tags we want to fetch from the API */
+enum class TextModerationTags {
```

Comment:
Question: Is there a way to use this API also for images? This might be useful in the future for some features (in particular, adding a picture to a given park)

Reply:
No, there's google api vision that we could use though. It would be a great addition, I could start working on it at the end of the week.